tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/DataType/TinyMCE.spec.ts

```diff
@@ -0,0 +1,279 @@
+import {test} from "@umbraco/playwright-testhelpers";
+import {expect} from "@playwright/test";
+
+const tinyMCEName = 'TestTinyMCE';
+
+test.beforeEach(async ({umbracoUi, umbracoApi}) => {
+  await umbracoApi.dataType.ensureNameNotExists(tinyMCEName);
+  await umbracoUi.goToBackOffice();
+  await umbracoUi.dataType.goToSettingsTreeItem('Data Types');
+});
+
+test.afterEach(async ({umbracoApi}) => {
+  await umbracoApi.dataType.ensureNameNotExists(tinyMCEName);
+});
+
+test('can create a rich text editor tinyMCE', {tag: '@smoke'}, async ({umbracoApi, umbracoUi}) => {
+  //  Arrange
+  const tinyMCELocatorName = 'Rich Text Editor [TinyMCE]';
+  const tinyMCEFilterKeyword = 'Rich Text Editor';
+  const tinyMCEAlias = 'Umbraco.RichText';
+  const tinyMCEUiAlias = 'Umb.PropertyEditorUi.TinyMCE';
+
+  // Act
+  await umbracoUi.dataType.clickActionsMenuAtRoot();
+  await umbracoUi.dataType.clickCreateButton();
+  await umbracoUi.dataType.clickNewDataTypeThreeDotsButton();
+  await umbracoUi.dataType.enterDataTypeName(tinyMCEName);
+  await umbracoUi.dataType.clickSelectAPropertyEditorButton();
+  await umbracoUi.dataType.selectAPropertyEditor(tinyMCELocatorName, tinyMCEFilterKeyword);
+  await umbracoUi.dataType.clickSaveButton();
+
+  // Assert
+  await umbracoUi.dataType.isSuccessNotificationVisible();
```

Comment:
We should use the '.doesSuccessNotificationHaveText(NotificationConstantHelper.success.saved);'